RecoParticleFlow/PFClusterProducer/plugins/LocalMaximumSeedFinder.h

```diff
@@ -19,7 +20,10 @@ class LocalMaximumSeedFinder final : public SeedFinderBase {
   const int _nNeighbours;
 
   const std::unordered_map<std::string,int> _layerMap;
-  std::array<std::pair<double,double>, 35> _thresholds;
+
+  typedef std::tuple<std::vector<int>, std::vector<double> , std::vector<double> > _i3tuple;
```

Comment:
capitalization and no leading underscore

Reply:
the no capitalization and leading underscore keep this variable compliant to the full set of variables inside the RecoParticleFlow/PFClusterProducer.
I will try to see if I can avoid to have this typedef

Reply:
this is not a variable, this is a typedef.
there is no example in this file of a type name using a lower case and an leading underscore.